test/phylogenies.js

```diff
@@ -0,0 +1,143 @@
+/*
+ * Test phylogenies.
+ */
+
+/* eslint-env mocha */
+
+// Load d3 as a global variable so it can be accessed by both phylotree.js (which
+// needs to add additional objects to it) and phyx (which needs to call it).
+global.d3 = require('d3');
+
+if (!Object.prototype.hasOwnProperty.call(global.d3, 'layout')) {
+  global.d3.layout = {};
+}
+require('../lib/phylotree.js/phylotree.js');
+
+const chai = require('chai');
+const phyx = require('../js/phyx');
+
+const { assert } = chai;
+
+describe('PhylogenyWrapper', function () {
+  describe('#constructor', function () {
+    it('should wrap a blank object', function () {
+      assert.isOk(new phyx.PhylogenyWrapper({}));
+    });
+
+    it('should be able extract all node labels in a phylogeny', function () {
```

Comment:
Are you testing the constructor here or the methods that are actually used? It seems like the latter, even if the internal extraction mechanism _currently_ all takes place in the constructor.

Reply:
You're right! I've moved this into a '#getNodeLabels' section.